### DISCUSSION TITLE
Few questions about deployment and layout for a dashboard

### DISCUSSION THREAD

**Discussion by bkxndata** [zauberzeug/nicegui#259] [Q&A] [answered]
Looks pretty sweet - considering using this to build a web app / dashboard.  Few questions:

- how easy it to deploy on aws / azure?  is there a tutorial somewhere.  while its cool to do locally, ultimately the real value is deploying so stakeholders can use it (think shinyapps.io for r shiny or streamlit cloud for streamlit)
- is there any better documentation on page layouts - how do i contain highchart objects in a grid, etc?

**Comment by rodja** [comment, marked as answer]
True, deploying to the cloud is very important. We have written a little bit [about deployment in the README.md](https://github.com/zauberzeug/nicegui#deployment). It's mainly about using the docker image. This should be fairly simple to deploy to aws / azure. Does that already answer your question? Or do you need more specific or docker-less hints.

Could you create a separate discussion for your question regarding page layouts? Then it would be easier for other users to find and follow up on it.
